bedrock/mozorg/templates/mozorg/home/home-a.html

```diff
@@ -0,0 +1,417 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import lazy_image with context %}
+
+{% extends "base-pebbles.html" %}
+
+{% add_lang_files "mozorg/home/index-quantum" %}
+
+{% block gtm_page_id %}data-gtm-page-id="Homepage"{% endblock %}
+
+{% block page_title %}{{ _('Internet for people, not profit') }}{% endblock %}
+
+{%- block page_title_suffix -%}
+  {% if LANG == 'en-GB' %}
+    — Mozilla (UK)
+  {% else %}
+    — Mozilla
+  {% endif %}
+{%- endblock -%}
+
+{% block page_desc %}
+  {{ _('Did you know? Mozilla — the maker of Firefox — fights to keep the Internet a global public resource, open and accessible to all.') }}
+{% endblock %}
+
+{% block body_id %}home{% endblock %}
+
+{% block extra_meta %}
+<!-- validates bing webmaster tools -->
+<meta name="msvalidate.01" content="B7B177115A634927D608514DA17B2574">
+<!-- YouTube Verification -->
+<meta name="google-site-verification" content="U9a6gH32vLIykvntaDToj-ytYhlZ1AfAgVEKstixQIE">
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'home-2018' %}
+{% endblock %}
+
+{% block site_header %}
+  {% if LANG.startswith('en-') %}
+    {% include 'includes/global-nav.html' %}
+  {% else %}
+    {% with is_home = True %}
+      {% set logo_src = static('img/pebbles/moz-wordmark-dark-reverse.svg') %}
+      {% include 'mozorg/home/includes/nav.html' %}
+    {% endwith %}
+  {% endif %}
+{% endblock %}
+
+{% macro card_block(class, tag_label, tag_class, title, ga_title, desc, image_url, image_highres_url, video_url, aspect_ratio, link_url, small_cta, meta) -%}
+<section class="card {% if class %}{{ class }}{% endif %} {% if video_url %}has-video{% endif %}">
+  <a class="card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link">
```

Comment:
Since we don't know if a card will always link off-site, adding this attribute would likely mean adding another parameter to the macro - which I'm trying to avoid unless really necessary. Adding `rel="external"` doesn't really have any major benefits (afaict), so I think we're OK to leave it.

Reply:
Agreed 👍